projects/plugins/jetpack/extensions/blocks/subscriptions/panel.js

```diff
@@ -164,6 +216,18 @@ export default function SubscribePanels() {
 						) }
 					</InspectorNotice>
 				) }
+				{ ! isModuleActive && ! isLoadingModules && (
+					<SubscriptionsPanelPlaceholder>
+						<Button
+							disabled={ isModuleActive || isChangingStatus }
+							isBusy={ isChangingStatus }
+							onClick={ enableSubscriptionsModule }
+							variant="secondary"
+						>
+							{ __( 'Activate Subscriptions', 'jetpack' ) }
+						</Button>
+					</SubscriptionsPanelPlaceholder>
+				) }
```

Comment:
I don't think we need to show 'Activate Subscriptions' in the post-publish panel. As @jeherve pointed out, doing so opens the door for a confusing user experience: activating Subscriptions won't send the already-published post.